brevbaker-api-model-common/src/main/kotlin/no/nav/pensjon/brevbaker/api/model/LetterMarkupImpl.kt

```diff
@@ -0,0 +1,136 @@
+package no.nav.pensjon.brevbaker.api.model
+
+import no.nav.brev.InterneDataklasser
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.Block
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.Block.Paragraph
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.Block.Title1
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.Block.Title2
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.Block.Type
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.ParagraphContent
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.ParagraphContent.Form.MultipleChoice
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.ParagraphContent.ItemList
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.ParagraphContent.Table
+import no.nav.pensjon.brevbaker.api.model.LetterMarkup.ParagraphContent.Text.FontType
+
+@Suppress("unused")
+@InterneDataklasser
+data class LetterMarkupImpl(
```

Comment:
Bare for sikkerhetsskyld, har du testet å åpne et brev som er lagret med de gamle klassene med det nye oppsettet i skribenten?